### ISSUE TITLE
Remove or update `self` link from feed.

### ISSUE THREAD

**Issue by kevincox** [pictuga/morss#75]
It seems that currently links in the feed are preserved. This is likely a good idea in general but creates problems with the `self` link.

- The self link should point to the feed itself. Since the feed URL is now on morss the self link should point there.
- The self link is used for WebSub to subscribe. If the original is preserved it may be used to subscribe in which case the original non-full-text entries would be returned to the reader. (The `hub` link should probably also be wiped out otherwise readers will attempt to subscribe to WebSub which won't work).

In conclusion I think the following should be done:

- The `self` link should be updated to be the morss link.
- Any `hub` links should be removed.

If preserving a link to the original is desired it would probably be best to include that as a `canonical` link. But I don't think this provides much value.

**Comment by pictuga** [comment]
Sounds like a fair point.
* `self`: doable but current feed parser does not address self link, so it'll have to be implemented beforehand
* `hub`: any chance you could share such an example feed with `hub` so that I can see what it looks like?

**Comment by kevincox** [comment]
The `hub` is just a `<link rel="hub" href="...">` in the atom namespace. It can appear in both Atom and RSS feeds. Some examples:

- https://kevincox.ca/feed.atom in Atom
- https://mozillagfx.wordpress.com/feed/ in RSS.
